jsonapi.gemspec

```diff
@@ -15,6 +15,8 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'jsonapi-parser', version
   spec.add_dependency 'jsonapi-renderer', version
 
+  spec.add_development_dependency 'virtus'
+  spec.add_development_dependency 'jsonapi-serializable'
```

Comment:
It seems wrong that this gem depends on `jsonapi-serializable`. If anything, it should be the other way around. In particular, it means that for bare `jsonapi-renderer` tests, we'll have to implement the classes manually, but I'll take care of it.